hudi-client/hudi-client-common/src/test/java/org/apache/hudi/client/transaction/TestConcurrentSchemaEvolutionTableSchemaGetter.java

```diff
@@ -278,19 +262,16 @@ private void addCommitOrDeltaCommitWithSchema(HoodieTableType tableType, String
 
   private static Stream<Arguments> commonTableConfigTestDimension() {
     return Stream.of(
-      // version 6 or 8, tableType
-      Arguments.of(true, HoodieTableType.COPY_ON_WRITE),
-      Arguments.of(true, HoodieTableType.MERGE_ON_READ),
-      Arguments.of(false, HoodieTableType.COPY_ON_WRITE),
-      Arguments.of(false, HoodieTableType.MERGE_ON_READ)
+      Arguments.of(HoodieTableType.COPY_ON_WRITE),
+      Arguments.of(HoodieTableType.MERGE_ON_READ)
     );
   }
 
   @ParameterizedTest
   @MethodSource("commonTableConfigTestDimension")
-  void testGetTableAvroSchemaInternalNoSchemaFoundEmptyTimeline(boolean enableMetadata, HoodieTableType tableType) throws IOException {
+  void testGetTableAvroSchemaInternalNoSchemaFoundEmptyTimeline(HoodieTableType tableType) throws IOException {
     // Don't set any schema in commit metadata or table config
-    initMetaClient(enableMetadata, tableType);
```

Comment:
`enableMetadata`: just realized that this is not for MDT but rather table version.  Since we only want to support table version 8, this is removed.

Reply:
In the write config, we may want to mention that the schema conflict resolution only works on table version 8.